lib/broccoli/ember-app.js

```diff
@@ -1845,6 +1845,30 @@ function addOutputFile(container, assetPath, options) {
     container[outputFile] = [];
   }
 
+  if (~container[outputFile].indexOf(assetPath)) {
```

Comment:
It's hard to get used to because it's not a general trick; it pretty much only works with `indexOf`